envpool/sokoban/level_loader.cc

```diff
@@ -1,5 +1,7 @@
 #include "level_loader.h"
 
+#include <pybind11/iostream.h>
```

Comment:
Why do you need this as opposed to `<iostream>`? Is it so that `StdCapture` below works?